README.md

```diff
@@ -100,6 +101,17 @@ Quick links:
 - [Sign the CLA to Create a Pull Request](https://github.com/sendgrid/python-http-client/blob/master/CONTRIBUTING.md#cla)
 - [Improvements to the Codebase](https://github.com/sendgrid/python-http-client/blob/master/CONTRIBUTING.md#improvements-to-the-codebase)
 
+<a name="local_setup"></a>
+
+# Local Setup of the project
+
+The simplest local development workflow is by using docker.
+
+> Steps
+
+1. Install Docker
+2. Run `docker-compose up` (This runs tests by default)
```

Comment:
I believe there should be a `docker-compose build` step as #2, can you please add that?

Reply:
Also, I LOVE  that the build runs the tests. That's awesome. 
It would be cool to make a note in the README about changing the run.sh file to another script if they want to run that.